examples/taxi-ride-akka/taxi-ride-pipeline/src/main/blueprint/blueprint.conf.txt

```diff
@@ -0,0 +1,25 @@
+blueprint {
```

Comment:
remove this file?

Reply:
I have 2 blueprints for two different use cases